### ISSUE TITLE
V sync

### ISSUE THREAD

**Issue by Algorythm4** [spoorn/sodium-forge#194]
Hello, 
Little issue on my Mac with the new m1 chip, since the beginning it's been working perfectly with the v-sync, which keeps the game at a steady 60 fps, and pretty fluid, but since yesterday, the v-sync is not working anymore and I don't know why, the only difference compared to before is that I installed macOS Monterey, but I don't know how it could make a difference. So now I can't play at all, because it's not fluid, I can reach 120 fps, but it also collapse to 30 fps some times, and it's not playable.

So in summary, the v-sync is not working, so I would like some help to play again.

Algorythm
